modules/core/src/loaders/xviz-websocket-loader.js

```diff
@@ -102,7 +103,7 @@ export default class XVIZWebsocketLoader extends XVIZLoaderInterface {
               message: message.data,
               onResult: this.onXVIZMessage,
               onError: this.onError,
-              debug: this._debug.bind('parse_message'),
+              debug: this.options.debug || this._debug.bind('parse_message'),
```

Comment:
I think we still want to send the 'parse_message' on the *_debug* object regardless of *options.debug*